tests/integrations/tests/integrations/test_openai.py

```
@@ -1054,3 +1095,470 @@ def test_30_embedding_usage_tracking(self, openai_client, test_config):
+            "function" in evt or "tool" in evt for evt in event_types
+        )
+
+        # Either should have tool calls detected or tool-related events
+        assert tool_calls_detected or has_tool_events, (
+            f"Should detect tool calls in streaming. "
+            f"Tool calls detected: {tool_calls_detected}, "
+            f"Event types: {list(event_types.keys())}"
+        )
+
+    @skip_if_no_api_key("openai")
+    def test_38_responses_reasoning(self, openai_client, test_config):
```

Ruff (0.14.2)

Unused method argument: `test_config`

(ARG002)